base/MAPL_LocstreamRegridder.F90

```diff
@@ -102,8 +102,8 @@ end subroutine regrid_2d_real32
 
    subroutine regrid_3d_real32(this,q_in,q_out, rc)
       class(LocstreamRegridder), intent(inout) :: this
-      real(kind=REAL32), intent(in) :: q_in(:,:,:)
-      real(kind=REAL32), intent(out) :: q_out(:,:)
+      real(kind=REAL32), intent(in) :: q_in(:,:,:)   ! (nx, ny, nz)
+      real(kind=REAL32), intent(out) :: q_out(:,:)   ! (nstation, nz)
       integer, optional, intent(out) :: rc
```

Comment:
Are these comments to the side always accurate?   Or are they particular to the StationSampler?     Seems like at least the 2nd line comment does not belong here.